stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamDimens.kt

```diff
@@ -37,6 +37,7 @@ import androidx.compose.ui.unit.dp
  * @param userReactionItemWidth The width of user reaction item.
  * @param userReactionItemAvatarSize The size of a user avatar in the user reaction item.
  * @param userReactionItemIconSize The size of a reaction icon in the user reaction item.
+ * @param headerElevation The elevation of the header.
```

Comment:
I would make this a bit more detailed - e.g. `The elevation of headers such as the ones that appear on the Channel or Message screens.`

Reply:
"The shape of the headers, such as the one appearing on the channels or messages screens."

"The elevation of the headers, such as the ones appearing on the channels or messages screens."

Mostly sending this for comma validation so that we don't have to make another commit for such a small thing

Reply:
Looks good with that comma